build/yamls/audit-webhook.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: audit-pv-volume
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi 
+  accessModes:
+    - ReadWriteMany
+  hostPath:
+    path: "/home/vagrant/audit"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: audit-pv-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 4Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: audit-deployment
+  labels:
+    app: audit
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: audit
+  template:
+    metadata:
+      labels:
+        app: audit
+    spec:
+      volumes:
+      - name: audit-pv-storage
+        persistentVolumeClaim:
+          claimName: audit-pv-claim
+      containers:
+      - name: audit
+        args: ["-d", "/data", "-v", "2"]
+        env:
+        - name: SERVICEACCOUNT_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.serviceAccountName
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: stanleywbwong/audit-webhook
```

Comment:
I'll take as an AI to move this to the antrea docker account, but this is on me.
The current image is fine for now.